app/services/twilio_service.rb

```diff
@@ -1,71 +1,75 @@
 require 'typhoeus/adapters/faraday'
 
-class TwilioService
-  cattr_accessor :telephony_service do
-    Twilio::REST::Client
-  end
+module TwilioService
```

Comment:
Thoughts on whether this should be moved to `twilio_services/base.rb`?

Reply:
Also, I'm not sure the Twilio Service makes as much sense for a superclass for the `Request` / `Response`, since it has a lot of concerns that extend beyond handling those and it seems a bit confusing that they are inherited.